hmda/src/test/scala/hmda/validation/rules/lar/quality/_2022/Q659_1Spec.scala

```diff
@@ -0,0 +1,28 @@
+package hmda.validation.rules.lar.quality._2022
+
+import hmda.model.filing.lar.LarGenerators._
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class Q659_1Spec extends LarEditCheckSpec {
+  override def check: EditCheck[LoanApplicationRegister] = Q659_1
+
+  val invalidExemptCodes = List("888", "88888")
+  property("Exempt field should not have a value of 888, or 88888") {
```

Comment:
@lchen-2101  this should be sprinkled with scenarios in which is passes the validation check